dockerfile/cuda11.1.1.dockerfile

```diff
@@ -42,39 +35,47 @@ RUN mkdir -p /root/.ssh && \
     sed -i "s/[# ]*Port.*/Port 22/" /etc/ssh/sshd_config && \
     echo "PermitUserEnvironment yes" >> /etc/ssh/sshd_config
 
-# Install OpenMPI
-ARG OMPI_VERSION=4.0.5
-RUN cd /tmp && \
-    wget -q https://www.open-mpi.org/software/ompi/v4.0/downloads/openmpi-${OMPI_VERSION}.tar.gz && \
-    tar xzf openmpi-${OMPI_VERSION}.tar.gz && \
-    cd openmpi-${OMPI_VERSION} && \
-    ./configure --enable-orterun-prefix-by-default && \
-    make -j all && \
-    make install && \
-    ldconfig && \
-    rm -rf /tmp/openmpi-${OMPI_VERSION}*
-
```

Comment:
Why remove OpenMPI?

Reply:
> remove duplicate cmake and ompi

it's already in description

Reply:
I mean the reason.

Reply:
it's already in base image and duplicate here, updated the description